getting-started-guides/csp/gcp/gcp.md

```diff
@@ -1,78 +1,105 @@
-# Get Started with XGBoost4J-Spark on Google Cloud
+# Get Started with XGBoost4J-Spark on GCP
 
-This is a getting started guide to XGBoost4J-Spark on Google Cloud. At the end of this guide, the reader will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on Google Cloud.
+This is a getting started guide to XGBoost4J-Spark on [Google Cloud Dataproc](https://cloud.google.com/dataproc). At the end of this guide, the reader will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on Google Cloud.
 
 ### Install & Cluster Spin up
 
-Then, use the steps from this [blog](https://medium.com/rapids-ai/scale-out-rapids-on-google-cloud-dataproc-8a873233258f) to create a RAPIDS cluster on Dataproc. Specifically, run through the steps in the Install Google Cloud SDK and the Create a Dataproc cluster sections.
-
-Here’s an example command:
-
-```
-CLUSTER_NAME=raghav-cluster
-ZONE=us-east1-c
-STORAGE_BUCKET=rapidsai-test-1
-NUM_GPUS_IN_MASTER=4
-NUM_GPUS_IN_WORKER=4
-NUM_WORKERS=2
-DATAPROC_BUCKET=dataproc-initialization-actions
-
-gcloud beta dataproc clusters create $CLUSTER_NAME  \
---zone $ZONE \
---master-accelerator type=nvidia-tesla-t4,count=$NUM_GPUS_IN_MASTER \
---master-machine-type n1-standard-32 \
---worker-accelerator type=nvidia-tesla-t4,count=$NUM_GPUS_IN_WORKER \
---worker-machine-type n1-standard-32 \
---num-workers $NUM_WORKERS \
---bucket $STORAGE_BUCKET \
---metadata "JUPYTER_PORT=8888" \
---initialization-actions gs://$DATAPROC_BUCKET/rapids/rapids.sh \
---optional-components=ANACONDA,JUPYTER \
---enable-component-gateway
-```
-
-This cluster should now have NVIDIA drivers and the CUDA run time.
+Prerequisites
+-------------
+* Apache Spark 2.3+
+* Hardware Requirements
+  * NVIDIA Pascal™ GPU architecture or better (V100, P100, T4 and later)
+  * Multi-node clusters with homogenous GPU configuration
+* Software Requirements
+  * NVIDIA driver 410.48+
+  * CUDA V10.0/9.2
```

Comment:
Now we support CUDA 10.1 also.
Better to add.

Reply:
will add when received confirmation